Wikipedia/assets/codemirror/config/codemirror-config-skr.json

```diff
@@ -0,0 +1,6 @@
+{"extCodeMirrorConfig":{"pluginModules":[],"tagModes":{"ref":"text/mediawiki","pre":"mw-tag-pre","nowiki":"mw-tag-nowiki"},"tags":{"pre":true,"nowiki":true,"gallery":true,"indicator":true,"timeline":true,"hiero":true,"charinsert":true,"ref":true,"references":true,"inputbox":true,"imagemap":true,"source":true,"syntaxhighlight":true,"poem":true,"categorytree":true,"section":true,"score":true,"templatestyles":true,"templatedata":true,"math":true,"ce":true,"chem":true,"graph":true,"maplink":true,"mapframe":true},"doubleUnderscore":[{"__notoc__":"notoc","__nogallery__":"nogallery","__forcetoc__":"forcetoc","__toc__":"toc","__noeditsection__":"noeditsection","__notitleconvert__":"notitleconvert","__notc__":"notitleconvert","__nocontentconvert__":"nocontentconvert","__nocc__":"nocontentconvert"},{"__NEWSECTIONLINK__":"newsectionlink","__NONEWSECTIONLINK__":"nonewsectionlink","__HIDDENCAT__":
```

Comment:
I'm curious as to how these were added. It probably only helps - perhaps these are auto-generated and they've been missing since the last time we ran a carthage update?

Reply:
You're correct - this happened when updating carthage.